app/views/attachments/_form.html.erb

```diff
@@ -14,13 +18,15 @@
       <% end %>
     </li>
     <li>
-      <div>
-        <%= f.check_box :released, help_text: "Checking this box will release the attachment to students" %>
-      </div>
+      <%= f.datetime_select :release_at, help_text: "Release attachment after this time" %>
```

Comment:
Is it possible to add text after this that will let you know whether or not the attachment will be released based on what date the user inputted?

So potentially instead of:
<img width="1061" alt="image" src="https://github.com/autolab/Autolab/assets/66969001/af66eea8-13cb-4311-9b55-0559acc36962">

It would look maybe something like:
(pretend the date has already passed or something so it's going to be released :sob:)
<img width="914" alt="image" src="https://github.com/autolab/Autolab/assets/66969001/1a72e92f-ff21-40a7-a639-190e6b97bae1">

(if assignment will be unreleased)
<img width="554" alt="image" src="https://github.com/autolab/Autolab/assets/66969001/9cd87a21-b115-4da9-8236-4ce2ce4da3ba">

If you can find a better wording for the status message it would be great! But I think since we removed the checkbox it would be nice for users to somehow know, with clarity, whether or not their assessment will be released upon creation!

Also, another small UX thing I noticed: the date shows up as gray.
<img width="224" alt="image" src="https://github.com/autolab/Autolab/assets/66969001/1f06fa11-3c86-427f-bb6f-dfd7d4acf977">

I think this makes sense when the date is filled in with the default value. But when the user changes the date, I think it should be black, like the rest of the input fields?

Or, you could even just do gray for any dates in the past, black for dates in the future (so further letting the user know about the attachment released status upon creation)?

Reply:
The gray text would be out of the scope for this PR;
The reason why it is grayed is because of materialize's style for readonly fields. It is set to readonly because we are using flatpickr's "altInput" option, which allows us to show one version of a date to the user while internally using another format.

In a sense, this is correct behavior, since the field is indeed readonly. I don't think it's a major deal as of now.

Reply:
Added code to indicate if attachment will be released immediately / how long in the future
<img width="332" alt="Screenshot 2023-11-10 at 15 17 00" src="https://github.com/autolab/Autolab/assets/9074856/b3c4738c-fe00-4f9e-8db9-393657a3d749">
<img width="280" alt="Screenshot 2023-11-10 at 15 17 08" src="https://github.com/autolab/Autolab/assets/9074856/6956f576-4325-4fcb-852a-232d9ca70f7c">